<!--
  Works with rocon_gateway_hub/pirate.launch - uses a non-default hub
  name so we can test hubs in parallel. 
  
  This one has no auto-discovery mode. Instead it uses the typical default
  settings for a local ubuntu redis server.
-->
<launch>
  <node pkg="rocon_gateway" type="gateway.py" name="gateway" output="screen">
    <rosparam command="load" file="$(find rocon_gateway)/config/default.yaml" />
    <rosparam param="hub_uri">http://localhost:6379</rosparam>
  </node>
</launch>
